core/templates/components/question-directives/question-editor/question-editor.component.ts

```diff
@@ -112,6 +112,7 @@ angular.module('oppia').component('questionEditor', {
         }
         QuestionUpdateService.setQuestionStateData(
           ctrl.question, updateFunction);
+        $rootScope.$apply();
```

Comment:
Prefer `$rootScope.$applyAsync` here and elsewhere.

Reply:
Also, I don't understand why it is added here. Could you please explain?

Reply:
I added them as some tests were failing. Removed them.